build/apis/exchange-chain-x-chain-api.md

```diff
@@ -575,10 +575,74 @@ curl -X POST --data '{
 }
 ```
 
+### avm.export
+
+Send a non-AVAX from the X-Chain to the C-Chain.  
```

Comment:
This method is also used to send AVAX to the P-Chain, so we should reflect that here